src/components/TreeView/TreeViewNode.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import TreeViewExpand from './TreeViewExpand';
+import TreeViewIcon from './TreeViewIcon';
+import TreeViewTextAndIconWrapper from './TreeViewTextAndIconWrapper';
+import TreeViewIndents from './TreeViewIndents';
+
+class TreeViewNode extends Component {
+  constructor(props) {
+    super(props);
+
+    // A node can be set to be expanded by default
+    this.state = {
+      expanded:
+        (props.node.hasOwnProperty('state') && props.node.state.expanded) ||
+        false
+    };
+
+    this.toggleExpand = this.toggleExpand.bind(this);
```

Comment:
we have bind methods for these now...

import { bindMethods } from '../../common/helpers';